truss/templates/server.Dockerfile.jinja

```diff
@@ -66,24 +66,31 @@ RUN $PYTHON_EXECUTABLE /cache_warmer.py {{file}} {{repo}}
 {%- if data_dir_exists %}
 COPY ./{{config.data_dir}} /app/data
 {%- endif %}
+# Do live reload installs before copys for better caching on publish
+{%- if config.live_reload %}
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        nginx supervisor curl && \
+        rm -rf /var/lib/apt/lists/*
+COPY ./control /control
+RUN python3 -m venv /control/.env \
+    && /control/.env/bin/pip3 install -r /control/requirements.txt
+
+COPY ./control/proxy.conf /etc/nginx/conf.d/proxy.conf
+COPY ./control/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+{%- endif %}
 
 COPY ./server /app
 COPY ./{{ config.model_module_dir }} /app/model
 COPY ./config.yaml /app/config.yaml
-    {%- if config.live_reload %}
-COPY ./control /control
-RUN python3 -m venv /control/.env \
-    && /control/.env/bin/pip3 install -r /control/requirements.txt
-    {%- endif %}
 {% endblock %}
 
 {% block run %}
     {%- if config.live_reload %}
 ENV HASH_TRUSS {{truss_hash}}
-ENV CONTROL_SERVER_PORT 8080
+ENV CONTROL_SERVER_PORT 8091
 ENV INFERENCE_SERVER_PORT 8090
-ENV SERVER_START_CMD="/control/.env/bin/python3 /control/control/server.py"
-ENTRYPOINT ["/control/.env/bin/python3", "/control/control/server.py"]
+ENV SERVER_START_CMD /usr/bin/supervisord
+ENTRYPOINT ["/usr/bin/supervisord"]
```

Comment:
All processes are running as root user; that's somewhat suboptimal from a security perspective. Would be good to use Docker's `USER` directive to run the user code in user land, nginx under nginx user.

Reply:
Would that have implications on the running the image in k8s?